lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -97,6 +98,11 @@ class GoogleCloudOutput < BufferedOutput
     #   }
     config_param :label_map, :hash, :default => nil
 
+    # labels (specified as a JSON object) is a set of custom labels
+    # provided at configuration time. It allows users to inject extra
+    # environmental information into every message.
```

Comment:
mention that it is a map of label name -> label value and give an example (similar to label_map above).